src/Symfony/Component/HttpKernel/Tests/EventListener/LocaleListenerTest.php

```diff
@@ -38,7 +38,7 @@ public function testDefaultLocaleWithoutSession()
     public function testLocaleFromRequestAttribute()
     {
         $request = Request::create('/');
-        session_name('foo');
+        //        session_name('foo');
```

Comment:
commenting is a no-go. If we don't want this code anymore, it should be removed.

Reply:
I know that, I think it's like you said, all of this should be skipped on 7.2